tests/web/APIRest.php

```diff
@@ -347,7 +347,7 @@ public function testGetItemsCommonDBChild() {
       $ticketTMF = new \TicketTemplateMandatoryField();
 
       $tt_id = $ticketTemplate->add([
-         'entities_id' => 0,
+         'entities_id' => 1,
```

Comment:
You should not rely on a hardcoded value, but rather sue something like:
```php
$ent1 = getItemByTypeName('Entity', '_test_child_1', true);
```

Reply:
Done.